charts/nginx-website/Chart.yaml

```
@@ -4,5 +4,5 @@ appVersion: "1.0"
 description: A Helm chart for serving website with nginx
 name: nginx-website
 maintainers:
-- name: lemeurherve
-version: 0.2.0
+- name: infra-team
+version: 0.2.1
```

```suggestion
- email: jenkins-infra-team@googlegroups.com
  name: jenkins-infra-team
version: 0.3.0
```

- feat == not a patch ;)